src/main.cpp

```diff
@@ -1233,16 +1233,17 @@ int main(int Argc, char *Argv[]) {
     if (targets.size() > 1)
         g->isMultiTargetCompilation = true;
 
+    std::vector<const char *> Args;
+    Args.push_back("ispc (LLVM option parsing)");
+    Args.push_back("--enable-nontrivial-unswitch");
```

Comment:
Is `--enable-nontrivial-unswitch` equivalent of passing `true` to llvm::SimpleLoopUnswitchPass constructor?

Reply:
Yes

Reply:
Then I suggest pass it through constructor. Using of `llvm::cl` options is unintuitive and hidden, we use it only where there is no other way of passing options.

Reply:
Done